src/python/include/Simplex_tree_multi.h

```diff
@@ -0,0 +1,183 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2014 Inria        """
+ *
+ *
+ *    Modification(s):
+ * 		- 2022/11 David Loiseaux / Hannah Schreiber : added multify / flatten to interface standard simplextree.
+ *      - YYYY/MM Author: Description of the modification
+ */
+#ifndef SIMPLEX_TREE_MULTI_H_
+#define SIMPLEX_TREE_MULTI_H_
+
+#include <algorithm>
+#include <gudhi/Simplex_tree.h>
+#include "utils/box.h"
+#include "utils/line.h"
+
+
+
+namespace Gudhi {
+/** Model of SimplexTreeOptions.
+ *
+ * Maximum number of simplices to compute persistence is <CODE>std::numeric_limits<std::uint32_t>::max()</CODE>
+ * (about 4 billions of simplices). */
+
+using value_type = double;
+struct Simplex_tree_options_multidimensional_filtration {
+public:
+	typedef linear_indexing_tag Indexing_tag;
+	typedef int Vertex_handle;
+	typedef std::vector<value_type> Filtration_value;
+	typedef std::uint32_t Simplex_key;
+	static const bool store_key = true;
+	static const bool store_filtration = true;
+	static const bool contiguous_vertices = false;
+};
+
+using option_multi = Simplex_tree_options_multidimensional_filtration;
+using option_std = Simplex_tree_options_full_featured;
+bool operator<(const std::vector<value_type>& v1, const std::vector<value_type>& v2)
+{
+	bool isSame = true;
+	if (v1.size() != v2.size()) isSame = false;
```

Comment:
It may be useful when comparing a projection of a filtration onto a lower-dimensional multi-filtration, with another lower-dimensional filtration, without doing the projection, but that would be very niche, and wouldn't be implemented like that. We can remove this indeed.